dat/Sinclair - ZX 81.dat

```diff
@@ -1,6089 +1,3149 @@
-)
-
-game (
-  name "Escape From Manhattan (Computer Rentals)"
-  description "Escape From Manhattan (Computer Rentals)"
-  year "1983"
-  manufacturer "N. Taylor"
-  rom ( name "escapefrommanhattan.p" size 16102 crc 63F8023F md5 135B8483BB5B316C8744D824ACBB86C1 sha1 53799E5EA4DA7A6C5EBBA8537126F25EBBEB8285 )
-)
-
-game (
-  name "Escape from Shazzar (International Publishing and Software Inc.)"
-  description "Escape from Shazzar (International Publishing and Software Inc.)"
-  year "1983"
-  manufacturer "A. F. Nuttall"
-  rom ( name "EscapeFromShazzar.tzx" size 16376 crc 78075B95 md5 0854165D7CAA9F5D43203E5F9C999FA0 sha1 992DFCECE29B42C729BBCA05422BF090DC7C275C )
-)
-
-game (
-  name "Escape from Shazzar (International Publishing and Software Inc.)"
-  description "Escape from Shazzar (International Publishing and Software Inc.)"
-  year "1983"
-  manufacturer "A. F. Nuttall"
-  rom ( name "escapefromshazzar.p" size 16146 crc C587593D md5 B2DC9A81AC27D4BC267C8D4A0236ACB5 sha1 48FA28F6B637E59F48A58E618617A98F83FE6813 )
-)
-
-game (
-  name "Adventure D: Espionage Island (Sinclair Research)"
-  description "Adventure D: Espionage Island (Sinclair Research)"
-  year "1982"
-  manufacturer "Artic"
-  rom ( name "EspionageIsland.tzx" size 15734 crc 2BF38DCA md5 50A59A5A1CA60349959BA2283E4188F8 sha1 1AFAA1C3773C78F8F92941D95DE226A727CC8822 )
-)
-
-game (
-  name "Adventure D: Espionage Island (Sinclair Research)"
-  description "Adventure D: Espionage Island (Sinclair Research)"
-  year "1982"
-  manufacturer "Artic"
-  rom ( name "espionageisland.p" size 15504 crc 81DDF2B0 md5 24C456B967F1B621685579289A656CEB sha1 9C2DB35B667F12A155AAD2F2E5725B9FCF54E290 )
-)
-
-game (
-  name "Adventure D: Espionage Island (Artic)"
-  description "Adventure D: Espionage Island (Artic)"
-  manufacturer "Artic"
-  rom ( name "EspionageIsland(Artic).tzx" size 15734 crc 06A3887C md5 30754AA09608D83DEB1C0194C04B85CB sha1 7B36F0BCE5ECCB9FC3DC4FB553516E5E7F5ADABE )
-)
-
-game (
-  name "Adventure D: Espionage Island (Artic)"
-  description "Adventure D: Espionage Island (Artic)"
-  manufacturer "Artic"
-  rom ( name "espionageisland(artic).p" size 15504 crc AC8DF706 md5 348A6B4FBEC8E375DAF1BE03D1C674DF sha1 8AD3E1AE97A34A6F6B1E2F653AEED770BA33F01E )
-)
-
-game (
-  name "FUNdamentals of Math (Timex)"
-  description "FUNdamentals of Math (Timex)"
-  year "1983"
-  manufacturer "ATM"
-  rom ( name "FUNdamentalsOfMath.tzx" size 39312 crc 75ADE9A1 md5 FBE76752CE6FD550C78F7116E8472D37 sha1 0C1FD6D5034FA73E8D1D3DD525E088CD96971BD7 )
-)
-
-game (
-  name "Fast Load Save (Musamy Software)"
-  description "Fast Load Save (Musamy Software)"
-  year "1982"
-  manufacturer "Musamy Software"
-  rom ( name "FastLoadSave.tzx" size 11000 crc 939F03CB md5 BE316FADC3492E475E6D07BDF9AE310E sha1 616DFC2662D46F415277990BEBF0B09E187ED36E )
-)
-
-game (
-  name "The Fast One (Campbell Systems)"
-  description "The Fast One (Campbell
```

Comment:
I see multiple `rom` entries. in some of the `games`. RetrArch playlists don't really work with multiple entries. I'm honestly unsure how it reacts to it.... Does the core care if it picks up the .p or the .tzx?

Reply:
That I'm not sure of. I just fixed this according to what DAT managers would normally work with. The other option would be to do something like postfixing the game names with `[P]` or `[TZX]` to determine what one it is. That's not the worst option, though it does mean a lot more manual labor to fix the file.

Reply:
At the very least, I can update this to only fix the `:` issue for now. Most DAT managers can cope with duplicate set names.

Reply:
I found a reasonable way to deal with this, see the latest revision.